src/shared/install.c

```diff
@@ -61,6 +61,11 @@ typedef enum SearchFlags {
         SEARCH_FOLLOW_CONFIG_SYMLINKS = 2,
 } SearchFlags;
 
+typedef enum PresetState {
+        PRESET_ENABLED = 1,
+        PRESET_DISABLED = 2,
+} PresetState;
```

Comment:
I don't strictly need it. I just thought it was easier to read. Would you prefer a boolean (true for enable, false for disable)?